Resources/Prototypes/_Goobstation/Changeling/Objectives/changeling.yml

```
@@ -0,0 +1,64 @@
+  id: EscapeIdentityObjective
+  description: I need to escape on the evacuation shuttle. Undercover.
+  components:
+  - type: Objective
+    icon:
+      sprite: Objects/Magic/magicactions.rsi
+      state: blink
+  - type: ImpersonateCondition
+  - type: TargetObjective
+    title: objective-condition-escape-identity-title
+  - type: PickRandomPerson
+    needsOrganic: true # Don't pick IPCs.
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)